### ISSUE TITLE
Q: Does this work on a regular Kindle?

### ISSUE THREAD

**Issue by kelna** [mattzzw/kindle-clock#2]
Does the code work on a regular (not Paperwhite) Kindle?

**Comment by FrzMtrsprt** [comment]
I don't think so. The script only contains configs for K4NT, PW2 and PW3, and none of them work with my kindle 7 basic. I guess you'll need to edit the script by yourself.
